src/lib/Tab/LocationView/ContentTab.php

```diff
@@ -60,12 +68,21 @@ public function renderView(array $parameters): string
         $contentType = $parameters['contentType'];
         $fieldDefinitions = $contentType->getFieldDefinitions();
         $fieldDefinitionsByGroup = $this->groupFieldDefinitions($fieldDefinitions);
+        $contentLanguages = $content->versionInfo->languageCodes;
+
+        $languages = $this->loadLanguages(
+            function (Language $language) use ($contentLanguages) {
```

Comment:
Do we need to pass a callable here? Can't you put this in `loadLanguages` method directly?